sdjournal/read.go

```diff
@@ -188,6 +188,29 @@ process:
 	return
 }
 
+func (r *JournalReader) FollowAsync(until <-chan time.Time, recieve chan<- []byte) <-chan error {
```

Comment:
few things:
- please add a docstring
- `s/recieve/receive`
- probably need to add a note that the `until` channel MUST be closed or this will leak a goroutine